managers/cheat_manager.c

```diff
@@ -97,7 +97,7 @@ void cheat_manager_apply_cheats(void)
          core_set_cheat(&cheat_info);
       }
    }
-    runloop_msg_queue_push(msg_hash_to_str(MSG_APPLYING_CHEAT), 1, 180, true);
+    runloop_msg_queue_push(msg_hash_to_str(MSG_APPLYING_CHEAT), 1, 60, true);
```

Comment:
It looks fine to me. This message is not important. Can always hit the button again.